packages/botonic-core/index.d.ts

```diff
@@ -1,10 +1,19 @@
+type StringMatcher = RegExp| string | ((data: string) => boolean);
+
 export interface Route {
-  path: string;
-  payload?: RegExp| string | ((input: string) => boolean);
-  text?: RegExp | string | ((input: string) => boolean);
+  path: StringMatcher;
+  payload?: StringMatcher;
+  text?: StringMatcher;
+  type?: StringMatcher;
+  intent?: StringMatcher;
+  session?: (session: any) => boolean;
```

Comment:
Can we use the Interface Session instead of the any? Why this parameter is returning a bool?

Reply:
👏 
I didn't do it because I had Session in react. I moved it to core and now I use it